driver/pparse.mli

```diff
@@ -28,6 +28,19 @@ type 'a ast_kind =
 | Structure : Parsetree.structure ast_kind
 | Signature : Parsetree.signature ast_kind
 
+val clear_ppx : unit -> unit
+(** Forget about all installed rewriters: in-process and external ones *)
+
+val clear_in_process_ppxs: unit -> unit
+(** Forget about all installed in-process rewriters *)
+
+val disable_in_process_ppx: Longident.t -> unit
```

Comment:
This should probably be called `remove_in_process_ppx`; "disable" implies that it can be re-enabled but it's just removed entirely.